src/dispatch/plugins/base/manager.py

```diff
@@ -76,6 +76,7 @@ def register(self, cls):
                 required=cls.required,
                 multiple=cls.multiple,
                 description=cls.description,
+                enabled=getattr(cls, "enabled", False)
```

Comment:
I think this should be `enabled=cls.enabled` instead all plugins should have this attribute if they are using the plugin system. See: https://github.com/Netflix/dispatch/blob/develop/src/dispatch/plugins/base/v1.py#L62